scripts/karma.coffee

```diff
@@ -179,17 +198,8 @@ module.exports = (robot) ->
     else
       msg.send msg.random karma.selfDeniedResponses(msg.message.user.name)
 
-  robot.respond /karma give allowance$/i, (msg) ->
-    if not robot.auth.hasRole(msg.message.user,'admin')
-      msg.send "I can't let you do that..."
-    else 
-      karma.allowances = {}
-      msg.send "All right... everyone can play again..."
-
-  robot.respond /karma show allowance$/i, (msg) ->
-    for item, value of karma.allowances
-      user = robot.brain.userForId(item)
-      msg.send "#{user?.name}: #{value}"
+  robot.respond /karma history$/i, (msg) ->
```

Comment:
Verb usage should include "my" if just listing the message-sending user or allow for a username input similar to ooc list